azure-quantum/azure/quantum/problem.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
```

Comment:
this probably should go in the `quantum/optimization` folder too

Reply:
I moved it to the serialzition folder with the serialzition code